solarforecastarbiter/io/reference_observations/sfa_reference_sites.csv

```diff
@@ -244,3 +244,10 @@ interval_length,name,latitude,longitude,elevation,network_api_id,network_api_abb
 1,"RTC,FSEC,Baseline",28.405,-80.7709,10.27022838592529,1403,,Etc/GMT+5,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
 1,"RTC,NV,Baseline",36.0275,-114.9215,695.5903930664062,1423,,Etc/GMT+8,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
 1,"RTC,NREL,Baseline",39.7397,-105.174,1764.428344726562,1426,,Etc/GMT+7,https://developer.nrel.gov/docs/solar/pvdaq-v3/,NREL PVDAQ
+1,California ISO,37,-120,0,CISO-ALL,,America/Los_Angeles,https://www.eia.gov/opendata/,EIA
```

Comment:
Is all the data from EIA hourly or the default? In any case this should be 60 so that the net load observations have an interval label of 1hr

Reply:
and 60 for the rest of the EIA sites as well